### ISSUE TITLE
Inconsistent layout "Send Message To Volunteers" form.

### ISSUE THREAD

**Issue by tditiecher** [HTBox/crisischeckin#252]
While working on issue #26 I saw that the HTML of Volunteer/CreateMessage.cshtml is inconsistent compared to the other forms:

```html
@{
   var width250Attributes = new { htmlAttributes = new { style = "width: 250px;" } };
   var width500Attributes = new { htmlAttributes = new { style = "width: 500px;" } };
}
<p>@Html.LabelFor(m => m.Subject, width250Attributes)</p>
<p>@Html.EditorFor(m => m.Subject, width500Attributes)</p>
```

While the other forms use the following pattern:
```html
<div class="form-group">
   @Html.LabelFor(m => m.Name, "Disaster Name")
   @Html.TextBoxFor(m => m.Name, new { @class = "form-control" })
</div>
```

Is there a reason why this form is done differently or can this be made consistent using the form-group class and 'div' instead of 'p'?

**Comment by BillWagner** [comment]
There is not a good reason, and it would be great to have them all be consistent. Please make this change.

**Comment by BillWagner** [comment]
Fixed in pull request #250
